Final Legal Chatbot Files/malegislature_final_scrapper.py

```
@@ -0,0 +1,71 @@
+        full_text = content.text.strip()
+        
+        # Clean junk
+        lines = full_text.split("\n")
+        clean_lines = []
+        for line in lines:
+            if any(junk in line.lower() for junk in ["general laws", "skip to content", "part i", "part ii", "title", "chapter", "prev", "next", "print page", "search"]):
+                continue
+            clean_lines.append(line.strip())
+
+        cleaned_text = "\n".join(clean_lines)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)